### PR TITLE
Fix ut for manifest modifications

### PR DESCRIPTION
**Pull request by bettybao1209**
fix #1481 ut

### CODE REVIEW

File: tests/neo.UnitTests/SmartContract/UT_InteropService.cs [hunk 1]
```diff
@@ -44,17 +44,17 @@ public void Runtime_GetNotifications_Test()
 
                 // Add return
 
-                script.EmitPush(true);
+                script.EmitPush(OpCode.RET);
```
Comment (erikzhang) [inline tests/neo.UnitTests/SmartContract/UT_InteropService.cs]: `EmitPush` or `Emit`?

File: src/neo/VM/Helper.cs [hunk 2]
```diff
@@ -109,6 +109,9 @@ public static ScriptBuilder EmitPush(this ScriptBuilder sb, object obj)
         {
             switch (obj)
             {
+                case ContractParameter parameter:
+                    sb.EmitPush(parameter);
+                    break;
```
Comment (erikzhang) [inline src/neo/VM/Helper.cs]: Where is it used?

File: tests/neo.UnitTests/SmartContract/UT_Syscalls.cs [hunk 3]
```diff
@@ -283,16 +288,18 @@ public void System_Runtime_GetInvocationCounter()
...
             using (var script = new ScriptBuilder())
             {
-                script.EmitSysCall(InteropService.Contract.Call, contractA.ScriptHash.ToArray(), "dummyMain", 0);
-                script.EmitSysCall(InteropService.Contract.Call, contractB.ScriptHash.ToArray(), "dummyMain", 0);
-                script.EmitSysCall(InteropService.Contract.Call, contractB.ScriptHash.ToArray(), "dummyMain", 0);
-                script.EmitSysCall(InteropService.Contract.Call, contractC.ScriptHash.ToArray(), "dummyMain", 0);
+                var parameter = new Array { 0, 1 }.ToParameter();
+
+                script.EmitSysCall(InteropService.Contract.Call, contractA.ScriptHash.ToArray(), "dummyMain", parameter);
+                script.EmitSysCall(InteropService.Contract.Call, contractB.ScriptHash.ToArray(), "dummyMain", parameter);
```
Comment (Tommo-L) [inline tests/neo.UnitTests/SmartContract/UT_Syscalls.cs]: > Where is it used?

`EmitSysCall` need an array parameter.

File: tests/neo.UnitTests/SmartContract/UT_InteropService.cs (hunk 1 above)
Comment (shargon) [reply tests/neo.UnitTests/SmartContract/UT_InteropService.cs]: Should be Emit

Comment (erikzhang) [reply tests/neo.UnitTests/SmartContract/UT_InteropService.cs]: @bettybao1209 Can you fix it?

Comment (bettybao1209) [reply tests/neo.UnitTests/SmartContract/UT_InteropService.cs]: I have modified it, please review.

**Pull request merged**